lib/erlef/community/languages.ex

```diff
@@ -0,0 +1,146 @@
+    },
+    %{
+      name: "Elchemy",
+      link: "https://wende.github.io/elchemy/",
+      icon: "/images/community/languages/elchemy.svg",
+      logo: nil,
+      about:
+        "Elchemy lets you write simple, fast and quality type safe code while leveraging both the Elm’s safety and Elixir’s ecosystem. It puts an emphasis on both Elm and Elixir compatibility and human-readable output of the generated Elixir code"
+    },
+    %{
+      name: "Bragful",
+      link: "https://github.com/bragful/ephp",
+      icon: "/images/community/languages/ephp.svg",
+      logo: nil,
+      about:
+        "Bragful provides PHP over Erlang, interpreted and written in 100% Erlang. Bragful can easily be integrated into web site projects as a PHP extension language for your server, embedded system(s), or Erlang and Elixir applications."
+    },
+    %{
+      name: "Fika",
+      link: "https://github.com/fika-lang/fika",
+      icon: nil,
+      logo: "/images/community/languages/fika.svg",
+      about:
+        "Fika is a modern programming language for the web. It a is statically typed, functional and runs on the BEAM. Fika is designed to be a beginner-friendly, safe and productive language for building web applications with a delightful developer experience."
+    },
+    %{
+      name: "Gleam",
+      link: "https://gleam.run/",
+      icon: "/images/community/languages/gleam.svg",
+      logo: nil,
+      about:
+        "Gleam is a fast, friendly, and functional language for building safe, scalable systems all on top of the Erlang VM. It provides a powerful static type system to help find and prevent bugs at compile time, long before it reaches your users."
+    },
+    %{
+      name: "Hamler",
+      link: "https://www.hamler-lang.org/",
+      icon: nil,
+      logo: "/images/community/languages/hamler.svg",
+      about:
+        "Hamler is a strongly-typed language with compile-time typechecking and built-in support for concurrency and distribution and empowers industries to build the next generation of scalable, reliable, realtime applications"
+    },
+    %{
+      name: "Joxa",
+      link: "http://www.joxa.org/",
+      icon: nil,
+      logo: "/images/community/languages/joxa.svg",
+      about:
+        "Joxa is a small semantically clean, functional lisp. It is a general-purpose language encouraging interactive development and a functional programming style."
+    },
+    %{
+      name: "LFE (Lisp Flavored Erlang)",
+      link: "https://lfe.io/",
+      icon: "/images/community/languages/lfe.svg",
+      logo: nil,
+      about:
+        "LFE is a Lisp-2+ written for the Erlang VM, providing 100% compatibility with Core Erlang. LFE is a serious Lisp for building scalable, distributed applications -- like the Erlang software that powers 40% of the world's telecommunications."
```

Comment:
```suggestion
        "LFE is a Lisp-2+ written for the Erlang VM, providing 100% compatibility with Core Erlang. LFE is a serious Lisp for building scalable, distributed applications—like the Erlang software that powers 40% of the world's telecommunications."
```
if possible we might want to use em-dashes, and proper usage mandates no spaces around them.